NMS-12999: Move Provisiond adapter docs to Reference (#3693)
OpenNMS/opennms: docs/modules/reference/pages/provisioning/adapters/rdns.adoc

[[adapter-rdns]]
= Reverse DNS

The DDNS adapter uses the dynamic DNS protocol to send updated PTR DNS records to your DNS system as nodes are provisioned into {page-component-title}.

> For every IP address found, there must be a valid reverse DNS zone created on your DNS server.

Author: User has to create it.  I'll update the text to make that more explicit.

> For every IP address found, there must be a valid reverse DNS zone created on your DNS server.

Author: What do you think about this?
```suggestion
For every IP address found, there must be a valid reverse DNS zone existing on your DNS server.
```

> For every IP address found, there must be a valid reverse DNS zone created on your DNS server.

Reviewer: Minor change:
```suggestion
For every IP address found, a valid reverse DNS zone must exist on your DNS server.
```

> For every IP address found, there must be a valid reverse DNS zone created on your DNS server.

Author: ```suggestion
For every IP address found, a corresponding reverse DNS zone must exist on your DNS server.
```
